lib/spatial_features/has_spatial_features/feature_import.rb

```diff
@@ -0,0 +1,50 @@
+require 'digest/md5'
+
+module SpatialFeatures
+  module FeatureImport
+    def update_features!(skip_invalid: false, **import_options)
```

Comment:
Make sure this method returns a truthy value if features were updated, and nil if they were not. It is used in both a save callback and in a boolean condition in connect engine.